src/components/home-page/Home.js

```diff
@@ -1,22 +1,36 @@
 import React, { useState } from "react";
 import { pickupData } from "../../mockData";
 import EventCard from "../eventCard/eventCard";
-import Container from "react-bootstrap/Container"
+import Container from "react-bootstrap/Container";
 import SportSelector from "../sportSelector/sportSelector";
+import { Col, Row } from "react-bootstrap";
 
 const Home = () => {
   const { users, sports } = pickupData;
   const defaultSport = sports[0];
   const [currentSport, setCurrentSport] = useState(defaultSport);
   const events = pickupData.events[currentSport];
   return (
-    <Container fluid="true">
-      <h1>Pickup Finder</h1>
-      <SportSelector sports={sports} currentSport={currentSport} setCurrentSport={setCurrentSport} />
-      {Object.entries(events).map(([eventId, event]) =>
-        <EventCard key={eventId} event={event} users={users} />
-      )}
+    <Container fluid="true" >
+      <Row>
+        <h1>Pickup Finder</h1>
+      </Row>
+      <Row className="pb-3">
```

Comment:
can you pass these as props to the Row component? in react-bootstrap, specifying styles via classNames is not recommended (this is a pattern seen in the regular bootstrap). if I'm wrong, let me know.

Reply:
(i.e. look at how style changes are passed as props for the card component https://react-bootstrap.github.io/components/cards/)

Reply:
are you suggesting to use style={{paddingBottom:"10px"}} or something like that? 

isn't assigning bootstrap classes to elements the whole appeal of using bootstrap?--ig i'm curious if you have a link that explains why specifying styles via boostrap classNames is not recommended...

Reply:
ah, seems like padding/margins need to be specified via classNames. 
but in react-bootstrap (as opposed to vanilla bootstrap) - in general - some styles can be applied by passing the right props to the component. for example, if you want to set the background of a card to "primary" you can pass the prop bg={"primary"}.